infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackService.kt

```diff
@@ -181,7 +182,10 @@ class LocationTrackService(
         val trackData =
             dao.getManyOfficialAtMoment(context.branch, ids, moment).let(::associateWithGeometries).map {
                 (track, geometry) ->
-                Triple(track, geometry, getGeocodingContext(track.trackNumberId))
+                // Deleted tracks are not validated for the context which might live on after track deletion
```

Comment:
Tämä on oikeastaan se ainoa juttu joka oikeasti on tarpeen tehdä: ei geokoodata raiteen päitä jos raide on poistettu. Huom, tässä ratanumero ei välttämättä ole poistettu joten tuo kontekstin luonnin esto ei ratkaise tätä casea, mutta ratkaisee saman ratanumero-puolella.